src/rusty/build-for-guix.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+rm -rf ${SCRIPT_DIR}/target
+mkdir ${SCRIPT_DIR}/target
+mkdir ${SCRIPT_DIR}/target/package
+
+cargo package --manifest-path ${SCRIPT_DIR}/sv2-ffi/Cargo.toml --allow-dirty
```

Comment:
I don't think Bitcoin Core build should ever call cargo for rust stuff - we should call rustc directly with the relevant libraries.

Reply:
I'm gonna remove the cargo dependency. And subtree all the rust sources.